solver/Cargo.toml

```diff
@@ -4,6 +4,8 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-anyhow = "1.0.33"
-ethcontract = { version = "0.8.0", default-features = false }
```

Comment:
ethcontract removed until we need it. I've changed the Address/H160 to primitive-types because I find it cleaner to use it directly.